net/microsocks/files/microsocks.init

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=95
+
+CONF="microsocks"
+
+start_service() {
+	config_load "$CONF"
+    
+	local _enabled
+	config_get _enabled "enabled" "enabled" "0"
+	[ "$_enabled" -eq "0" ] && exit 1
+    
+    	local _port
+	local _listenip
+	local _bindaddr
+	local _user
+	local _format
+	local _auth_once
+
+	config_get _port "config" "port"
+	config_get _listenip "config" "listenip"
+	config_get _bindaddr "config" "bindaddr"
+	config_get _user "config" "user"
+	config_get _password "config" "password"
+	config_get_bool _auth_once "config" "auth_once" 0
+    
+	procd_open_instance "$CONF"
+	procd_set_param command /usr/bin/microsocks
+	if [ ! -z "$_port" ]; then
+		procd_append_param command  -p ${_port}
+	fi
+	if [ ! -z "$_listenip" ]; then
+		procd_append_param command  -i ${_listenip}
+	fi
+	if [ ! -z "$_bindaddr" ]; then
+		procd_append_param command  -b ${_bindaddr}
+	fi
+	if [ ! -z "$_user" ]; then
+		procd_append_param command -u \"${_user}\"
+	fi
+	if [ ! -z "$_password" ]; then
+		procd_append_param command -P \"${_password}\"
+	fi
+	if [ "$_auth_once" -eq "1" ]; then
+		procd_append_param command -1
+	fi
+	
+	procd_set_param respawn
+	procd_set_param stderr 1
+	# TODO: Make it dependable on some verbose/debug config setting?
+	# procd_set_param stdout 1
+    
+	procd_close_instance
+}
+
+
+reload_service() {
+	stop
+	start
+}
```

Comment:
```suggestion
}

service_triggers() { 
	procd_add_reload_trigger "$CONF" 
}
```

`service_triggers` is required so procd can monitor uci changes and reload it accordingly.